jaggery-modules/rxt/module/scripts/asset/asset.js

```diff
@@ -1160,9 +1160,9 @@ var asset = {};
         var asset = this.get(id);
         var tagged; //Assume that the tag will not be applied
         var utilsAPI = require('utils');
-        //If the user has provided a single tag then it should be 
+        //If the user has provided a single tag then it should be
         //assigned to an array to keep the registry invocation uniform
-        if (!utilsAPI.reflection.isArray(tags)) {
+        if (typeof tags === 'string') {
```

Comment:
This either can be a single string or an array of strings.
Please refer : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof
http://stackoverflow.com/questions/27509/detecting-an-undefined-object-property